docs/manage-table-partition-using-gravitino.md

```diff
@@ -0,0 +1,370 @@
+| range          | A range partition represents a result of range [partitioning](./table-partitioning-bucketing-sort-order-indexes.md#table-partitioning).        |
+| list           | A list partition represents a result of list [partitioning](./table-partitioning-bucketing-sort-order-indexes.md#table-partitioning).          |
+
+For JSON examples:
+
+<Tabs groupId="partitions">
+<TabItem value="identity" label="identity">
+
+```json
+{
+  "type": "identity",
+  "name": "dt=2008-08-08/country=us",
+  "fieldNames": [
+    [
+      "dt"
+    ],
+    [
+      "country"
+    ]
+  ],
+  "values": [
+    {
+      "type": "literal",
+      "dataType": "date",
+      "value": "2008-08-08"
+    },
+    {
+      "type": "literal",
+      "dataType": "string",
+      "value": "us"
+    }
+  ]
+}
+```
+
+:::note
+The values must in the same order as the field names.
+
+When adding an identity partition to a Hive partitioned table, the specified partition name is disregarded. This is because Hive generates the partition name based on field names and values.
+:::
+
+</TabItem>
+<TabItem value="range" label="range">
+
+```json
+{
+  "type": "range",
+  "name": "p20200321",
+  "upper": {
+    "type": "literal",
+    "dataType": "date",
+    "value": "2020-03-21"
+  },
+  "lower": {
+    "type": "literal",
+    "dataType": "null",
+    "value": "null"
+  }
+}
+```
+
+</TabItem>
+<TabItem value="list" label="list">
+
+```json
+{
+  "type": "list",
+  "name": "p202204_California",
+  "lists": [
+    [
+      {
+        "type": "literal",
+        "dataType": "date",
+        "value": "2022-04-01"
+      },
+      {
+        "type": "literal",
+        "dataType": "string",
+        "value": "Los Angeles"
+      }
+    ],
+    [
+      {
+        "type": "literal",
+        "dataType": "date",
+        "value": "2022-04-01"
+      },
+      {
+        "type": "literal",
+        "dataType": "string",
+        "value": "San Francisco"
+      }
+    ]
+  ]
+}
+```
+
+:::note
+Each list in the lists must have the same length. The values in each list must correspond to the field definitions in the list [partitioning](./table-partitioning-bucketing-sort-order-indexes.md#table-partitioning).
+:::
+
+</TabItem>
+</Tabs>
+
+For Java examples:
+
+<Tabs groupId="partitions">
+<TabItem value="identity" label="Identity">
+
+```java
+Partition partition =
+        Partitions.identity(
+            "dt=2008-08-08/country=us",
+            new String[][] {{"dt"}, {"country"}},
+            new Literal[] {
+              Literals.dateLiteral(LocalDate.parse("2008-08-08")), Literals.stringLiteral("us")
+            },
+            Maps.newHashMap());
+```
+
+:::note
+The values are in the same order as the field names.
+
+When adding an identity partition to a Hive partitioned table, the specified partition name is disregarded. This is because Hive generates the partition name based on field names and values.
+:::
```

Comment:
Do you need to say again here?

Reply:
Yes, because the note block belongs different TabItem